hasher-matcher-actioner/webapp/src/pages/bank-management/ViewBankMember.tsx

```diff
@@ -169,6 +204,27 @@ export default function ViewBankMember(): JSX.Element {
           </Col>
         </Row>
       )}
+      <Modal show={showConfirmModal} onHide={() => setShowConfirmModal(false)}>
+        <Modal.Header closeButton>
+          <Modal.Title>Remove the BankMember </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <p>
+            Are you sure you want to remove this bank member? Once the index
+            rebuilds, HMA will stop matching against this member.
```

Comment:
nit: see nit above, context on how and if this can be undone (or if it can't) for the user to know.

Reply:
Done.. Here's what it looks like now
<img width="556" alt="Screen Shot 2021-12-21 at 16 07 39" src="https://user-images.githubusercontent.com/217056/146998137-5b3dee35-ad9a-45ec-88ba-76fc79043af0.png">
.